nbs/01a_losses.ipynb

```diff
@@ -62,7 +62,15 @@
     "class BaseLoss():\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.            "Used on predictions in inference"
</pre><p>My opinion would be to modify the doc string to describe what it does as well as what it acts on.  Maybe</p><p> </p><p><code>Converts raw predictions/probabilities to class predictions</code> or something like that?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastai-docment-sprint/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>